src/surface_conditions/surface_setups.jl

```diff
@@ -101,9 +101,10 @@ function (::Bomex)(params)
     q_vap = FT(0.02245)
     θ_flux = FT(8e-3)
     q_flux = FT(5.2e-5)
+    z0 = FT(1e-4)
```

Comment:
why not zero?

Reply:
You can't use zero as it is used in the denominator in surface flux. 1e-4 for roughness is a typical value over ocean.

Reply:
Or maybe it's a different variable than what I think it is/ Maybe a longer name?

Reply:
Right, it is roughness, and it is commonly noted as z0 in Monin-Obukhov.

Reply:
I can add a comment on the MoninObukhov struct to make this more clear, if that's helpful?